content/_guides/general-troubleshooting-guide.md

```diff
@@ -32,6 +32,7 @@ When it comes to Component problems, everything is quite simple. You can check w
 
 ## Platform Issues
 While the Platform itself is not error-prone, we surely can not predict every possible issue with custom Components and complex Flows. Check [this document](platform-behavior) for some details on the common Platform troubles and the general way to fix them.
+Most of the platform error should not lead to messages lost unless it will fail in the messaging subsystem itself. Platform use RabbitMQ to handle message transport between integration steps.
```

Comment:
Most of platform errors won't lead to message loss, unless the messaging subsystem itself fails. Platform uses RabbitMQ to handle message transport between integration steps.